server/chat-plugins/mafia.js

```diff
@@ -51,8 +51,8 @@
 
 /** @type {typeof import('../../lib/fs').FS} */
 const FS = require(/** @type {any} */('../../.lib-dist/fs')).FS;
-const LOGS_FILE = 'config/chat-plugins/mafia-logs.json';
-const BANS_FILE = 'config/chat-plugins/mafia-bans.json';
+const LOGS_FILE = '../../config/chat-plugins/mafia-logs.json';
+const BANS_FILE = '../../config/chat-plugins/mafia-bans.json';
```

Comment:
I accidentally added this but I couldn't run a PS server without it and even then it still crashed when ran. I'll go ahead and remove it and investigate what's actually happening later.